lib/galaxy/tool_util/verify/asserts/__init__.py

```diff
@@ -12,16 +12,19 @@
 # create a new module of assertion functions, create the needed python
 # source file "test/base/asserts/<MODULE_NAME>.py" and add
 # <MODULE_NAME> to the list of assertion module names defined above.
-assertion_modules = []
+assertion_functions = {}
 for assertion_module_name in assertion_module_names:
-    full_assertion_module_name = f"galaxy.tool_util.verify.asserts.{assertion_module_name}"
+    full_assertion_module_name = 'galaxy.tool_util.verify.asserts.' + assertion_module_name
```

Comment:
I did prefer the f-string, it's more readable and theoretically slightly faster

Reply:
```suggestion
    full_assertion_module_name = f"galaxy.tool_util.verify.asserts.{assertion_module_name}"
```